controllers/apply/form-router/index.js

```diff
@@ -64,10 +70,22 @@ function initFormRouter({
         next();
     }
 
+    function setFormBrand(req, res, next) {
+        const validBrand = allowedBrands.find(
+            brand => brand.slug === req.query.brand
+        );
+        if (validBrand) {
```

Comment:
I'd perhaps write this as:

```js

if (validBrand) {
  set(req.session, formBrandSessionKey(), validBrand);
  req.session.save(() => {
    next();
  });
} else {
  next();
}
```

So that we only need to save the session if touching it.